fuel-core/src/database.rs

```diff
@@ -263,15 +269,20 @@ impl InterpreterStorage for Database {
 
 #[async_trait]
 impl RelayerDb for Database {
```

Comment:
Could we move all this RelayerDb impl logic into a separate mod? Would help keep the main DB file more manageable.

Reply:
Makes sense, will do the same with VM related impl, and i would like to move db tables to separate submodules to group them together. https://github.com/FuelLabs/fuel-core/issues/385